[2025-04-03T10:14:56.097+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-03T10:14:56.135+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_name.upload_to_bigquery manual__2025-04-03T10:14:31.221732+00:00 [queued]>
[2025-04-03T10:14:56.154+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_name.upload_to_bigquery manual__2025-04-03T10:14:31.221732+00:00 [queued]>
[2025-04-03T10:14:56.154+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-03T10:14:56.176+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): upload_to_bigquery> on 2025-04-03 10:14:31.221732+00:00
[2025-04-03T10:14:56.189+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1388) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-03T10:14:56.192+0000] {standard_task_runner.py:72} INFO - Started process 1397 to run task
[2025-04-03T10:14:56.195+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_dag_name', 'upload_to_bigquery', 'manual__2025-04-03T10:14:31.221732+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/intermediaire_table.py', '--cfg-path', '/tmp/tmpl8hid69h']
[2025-04-03T10:14:56.197+0000] {standard_task_runner.py:105} INFO - Job 75: Subtask upload_to_bigquery
[2025-04-03T10:14:56.269+0000] {task_command.py:467} INFO - Running <TaskInstance: my_dag_name.upload_to_bigquery manual__2025-04-03T10:14:31.221732+00:00 [running]> on host d6badcb20287
[2025-04-03T10:14:56.516+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag_name' AIRFLOW_CTX_TASK_ID='upload_to_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T10:14:31.221732+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-03T10:14:31.221732+00:00'
[2025-04-03T10:14:56.519+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-03T10:14:56.520+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-03T10:14:56.520+0000] {logging_mixin.py:190} INFO - Current task name:upload_to_bigquery state:running start_date:2025-04-03 10:14:56.136220+00:00
[2025-04-03T10:14:56.521+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name and current dag run status:running
[2025-04-03T10:14:56.521+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-03T10:14:56.545+0000] {base.py:84} INFO - Retrieving connection 'gcp_conn'
[2025-04-03T10:14:56.555+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/intermediaire_table.py", line 70, in upload_to_bigquery
    hook.insert_rows(table=f"{DATASET_NAME}.{TABLE_NAME}", rows=df.to_dict(orient="records"))
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 276, in insert_rows
    raise NotImplementedError()
NotImplementedError
[2025-04-03T10:14:56.573+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-04-03T10:14:56.573+0000] {logging_mixin.py:190} INFO - Task start:2025-04-03 10:14:56.136220+00:00 end:2025-04-03 10:14:56.572321+00:00 duration:0.436101
[2025-04-03T10:14:56.574+0000] {logging_mixin.py:190} INFO - Task:<Task(_PythonDecoratedOperator): upload_to_bigquery> dag:<DAG: my_dag_name> dagrun:<DagRun my_dag_name @ 2025-04-03 10:14:31.221732+00:00: manual__2025-04-03T10:14:31.221732+00:00, state:running, queued_at: 2025-04-03 10:14:31.237779+00:00. externally triggered: True>
[2025-04-03T10:14:56.574+0000] {logging_mixin.py:190} INFO - Failure caused by 
[2025-04-03T10:14:56.574+0000] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=my_dag_name, task_id=upload_to_bigquery, run_id=manual__2025-04-03T10:14:31.221732+00:00, execution_date=20250403T101431, start_date=20250403T101456, end_date=20250403T101456
[2025-04-03T10:14:56.615+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-03T10:14:56.616+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 75 for task upload_to_bigquery (; 1397)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/intermediaire_table.py", line 70, in upload_to_bigquery
    hook.insert_rows(table=f"{DATASET_NAME}.{TABLE_NAME}", rows=df.to_dict(orient="records"))
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 276, in insert_rows
    raise NotImplementedError()
NotImplementedError
[2025-04-03T10:14:56.651+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-03T10:14:56.678+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-03T10:14:56.680+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
